db/migrate/20150912205745_create_seed_trades.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSeedTrades < ActiveRecord::Migration
+  def change
+    create_table :seed_trades do |t|
+      t.text       :message
+      t.text       :address
+      t.datetime   :requested_date
+      t.datetime   :accepted_date
+      t.datetime   :declined_date
```

Comment:
But is there any situation in which we would fill in _both_ `accepted_date` AND `declined_date`? If not, there's no need for two columns.